_doc/myst-parser.md

```diff
@@ -0,0 +1,109 @@
+# MyST parser - Markdown support in Sphinx
+
+To support Markdown-based documentation, Sphinx can use MyST-Parser.
+
+## Configuration and setup
+
+Documentation for this can be found at [https://www.sphinx-doc.org/en/master/usage/markdown.html](https://www.sphinx-doc.org/en/master/usage/markdown.html)
+
+If you haven't done so already, please follow the [instructions to build the training docu locally](https://training.plone.org/5/mastering-plone/about_mastering.html#building-the-documentation-locally)
+
+Inside your Sphinx project:
+
+- `pip install -—upgrade myst-parser`
+
+in conf.py:
+
+```py
+extensions = [
+  ...
```

Comment:
```suggestion
  # ...
```

Pygments will not apply syntax highlighting for the specified language when it has invalid syntax. Commenting the ellipsis makes it valid Python.